tools/reviewer/webapp/src/app/shared/services/auth.guard.ts

```diff
@@ -1,31 +1,15 @@
-import { AuthService } from '@/shared/services/auth.service';
 import { Injectable } from '@angular/core';
-import {
-  ActivatedRouteSnapshot,
-  CanActivate,
-  Router,
-  RouterStateSnapshot,
-} from '@angular/router';
+import { CanActivate } from '@angular/router';
+import { AngularFireAuth } from 'angularfire2/auth';
 import { Observable } from 'rxjs';
 
 @Injectable()
 export class AuthGuard implements CanActivate {
-  constructor(private router: Router, private authService: AuthService) { }
+  constructor(private angularFireAuth: AngularFireAuth) { }
 
-  canActivate(
-    route: ActivatedRouteSnapshot,
-    state: RouterStateSnapshot,
-  ): Observable<boolean> {
-    // There is a bug.
-    // If you open the app without internet connection,
```

Comment:
I tried this, and got the Chrome dinosaur, so now sure how to reproduce. Is this resolved now?

Reply:
To reproduce it you need to start app from source.
But bug is gone. Not by the update. Somewhere before, I'm not sure what is moment it's.